libs/SalesforceSDK/src/com/salesforce/androidsdk/push/PushMessaging.java

```diff
@@ -58,17 +64,13 @@ public class PushMessaging {
 	// Public constants.
     public static final String UNREGISTERED_ATTEMPT_COMPLETE_EVENT = "com.salesforce.mobilesdk.c2dm.UNREGISTERED";
     public static final String UNREGISTERED_EVENT = "com.salesforce.mobilesdk.c2dm.ACTUAL_UNREGISTERED";
-    public static final String ACCOUNT_BUNDLE_KEY = "account_bundle";
```

Comment:
These constants are no longer needed.  I just noticed they are public.  Is that relevant?  Their use seemed really specific to user account push notification registration.